src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+type PropsPagination = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}: PropsPagination) => {
+  const items = getNumbers(1, total);
+  const totalPages = Math.ceil(items.length / perPage);
+  const totalPagesArr = getNumbers(1, totalPages);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={cn(
+          'page-item',
+          { disabled: currentPage === 1 },
+          { active: currentPage === 1 },
+        )}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={currentPage === 1}
```

Comment:
The `aria-disabled` attribute is redundant here as you are already handling the disabled state in the onClick handler. You can safely remove it.

Reply:
It was literally written in the task conditions that we should use this attribute